### ISSUE TITLE
Popup open then close

### ISSUE THREAD

**Issue by profezzeur** [jasdumas/shinyGEO#77]
Hello,

I installed ShinGEO on my home server (with rstudio-server and shiny-server), but after allow popups from my subdomain, shinyGEO open then close immediatly.

When I repeat the process, I saw that a port is randomly set to shinyGEO : perhaps I have to fix it, but how to ? 

Any idea ?

Dans l'attente de vous lire

Bien à vous

**Comment by gdancik** [comment]
If _shinyGEO_ is opening then closing immediately, my best guess is that there is an error causing it to crash, most likely because a required library is missing. 

I would first make sure that _shinyGEO_ can be run locally (instead of from shiny-server). In fact I'm not sure there is a good reason to run _shinyGEO_ from a shiny server unless you want to provide network access. But if you are not aware then this is provided here: http://gdancik.github.io/shinyGEO/ and you can always submit a pull request if you've made changes.

To run locally you can follow the steps from the README here: https://github.com/gdancik/shinyGEO/
That version of _shinyGEO_ may also be more stable, so I suggest to work with that one. 

But to answer your questions on shiny server, the default port is 3838, but can be changed. If the port is set to shinyGEO then something is off.  You can specify the port in /shiny-server/config/default.config, make sure the 'listen' line says:

 listen 3838;

Also on shiny server, all packages must be installed to the system library, using something like:
sudo su - -c "R -e \"install.packages('shiny')\""

I hope this helps.

**Comment by profezzeur** [comment]
Hello,

First thanks for your explicit answer.

I don't use ShinyGEO for me (not yet !), I installed it on my home server for some psychiatrists in an another hospital than mine, and I can't access to their computer. 

It's working, but only when they launch ShinyGEO from RStudio-server with the url http://public-IP-of-my-server:8787 (RStudio-server default port) : the popup window don't disappear anymore, and now they can use your ShinyApp, so it's cool thanks !

I think I have to work with my nginx sites-availabled file and/or my "shinydashboard" UI (I created a default shinydashboard with 3838 port, and if ShinyGEO is accessible in the default menu it would be awesome..!).

Bien à vous
